src/java/edu/slu/tpen/servlet/CanvasServlet.java

```diff
@@ -7,16 +7,20 @@
 
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import static edu.slu.tpen.entity.Image.Canvas.getAnnotationListsForProject;
+import static edu.slu.tpen.entity.Image.Canvas.getLinesForProject;
```

Comment:
Nice, good catch.  We aren't going to do lines anymore though, so now this is unused.  Just remove it.